app/javascript/components/shared/Modal.jsx

```diff
@@ -3,34 +3,31 @@ import { Modal as BootstrapModal } from 'react-bootstrap';
 import PropTypes from 'prop-types';
 
 export default function Modal({
-  modalButton, title, body, footer,
+  modalButton, title, body,
 }) {
   const [show, setShow] = useState(false);
 
   const handleClose = () => setShow(false);
   const handleShow = () => setShow(true);
 
-  const Footer = React.cloneElement(footer, { handleClose });
```

Comment:
What's your reasoning for removing the footer?

Reply:
The modal template was too restrictive - it is not possible not make a one size fits all.
The footer doesn't bring any added value - it saves two or three CSS classes at most.
Removing the footer gives us more flexibility for the design.